src/pe_reports/report_generator.py

```diff
@@ -0,0 +1,42 @@
+"""A tool for creating Posture & Exposure reports."""
+
+# Standard Python Libraries
+import os
+import sys
+
+# Third-Party Libraries
+from _version import __version__
+from pages import Pages
```

Comment:
This needs to be a relative import.

```suggestion
from .pages import Pages
```

Reply:
Resolved in 689417d